design/sketch.webidl

```diff
@@ -437,15 +437,32 @@ enum WebGPUStoreOp {
     "store",
 }
 
-dictionary WebGPURenderPassAttachmentDescriptor {
+dictionary WebGPURenderPassColorAttachmentDescriptor {
     WebGPUTextureView attachment;
+
     WebGPULoadOp loadOp;
     WebGPUStoreOp storeOp;
+    float clearR;
```

Comment:
let's have a `WebGPUColor` type so that this is a single field. We'll need it in other places likely as well.